www/README.md

```diff
@@ -71,15 +71,15 @@ Like `npm start`, it runs `bower install` to install and/or update the framework
 To make the production code available to the Kubernetes api server, run this command from the top level directory:
 
 ```
-hack/build-ui.sh
+hack/build-ui.sh dashboard
 ```
 
-It runs the `go-bindata` tool to package the generated `app` directory and other user interface content, such as the Swagger documentation, into `pkg/ui/datafile.go`. Note: go-bindata can be installed with `go get github.com/jteeuwen/go-bindata/...`.
+It runs the `go-bindata` tool to package the generated `app` directory and other user interface content, such as the Swagger documentation, into `pkg/ui/data/dashboard/datafile.go`. Note: go-bindata can be installed with `go get github.com/jteeuwen/go-bindata/...`.
```

Comment:
This comment says that the swagger documentation is placed in the same package as the dashboard, which is incorrect.